modules/api/src/main/java/org/apache/ignite/table/Table.java

```diff
@@ -73,6 +75,16 @@
      */
     KeyValueBinaryView kvView();
 
+    /**
+     * Creates a transactional view of the table.
+     *
+     * @param tx The transaction.
+     * @return Transactional table view.
+     * @see Transaction
+     * @see IgniteTransactions
+     */
+    Table withTx(Transaction tx);
```

Comment:
I think we should have this method in all Views (KV and Table) because
user usually reuse cached Table object instance.

kvView.withTx(tx)  looks better, than table.withTx(tx).kvView()